0.6-dev/schemas/image.schema

```diff
@@ -0,0 +1,435 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://ngff.openmicroscopy.org/0.5/schemas/image.schema",
+  "title": "OME-Zarr Image",
+  "description": "The zarr.json attributes key",
+  "type": "object",
+  "properties": {
+    "ome": {
+      "description": "The versioned OME-Zarr Metadata namespace",
+      "type": "object",
+      "properties": {
+        "multiscales": {
+          "$ref": "#/$defs/multiscales"
+        },
+        "omero": {
+          "$ref": "#/$defs/omero"
+        },
+        "version": {
+          "$ref": "https://ngff.openmicroscopy.org/0.5/schemas/_version.schema"
+        }
+      },
+      "required": [
+        "multiscales",
+        "version"
+      ]
+    }
+  },
+  "required": [
+    "ome"
+  ],
+  "$defs": {
+    "multiscales": {
+      "description": "The multiscale datasets for this image",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "name": {
+            "type": "string"
+          },
+          "datasets": {
+            "type": "array",
+            "minItems": 1,
+            "items": {
+              "type": "object",
+              "properties": {
+                "path": {
+                  "type": "string"
+                },
+                "coordinateTransformations": {
+                  "$ref": "coordinate_transformation.schema"
+                }
+              },
+              "required": [
+                "path",
+                "coordinateTransformations"
+              ]
+            }
+          },
+<<<<<<< HEAD:latest/schemas/image.schema
```

Comment:
I've corrected these as I thought correct in https://github.com/joshmoore/ngff/commit/878e15c550b4b71a3b0d7e690ce64dd91d963ca6